### ISSUE TITLE
make veryclean ; make # Broken with EZFIO.cfg

### ISSUE THREAD

**Issue by scemama** [LCPQ/quantum_package#63]
```
cd src
make veryclean
make

...

IRPF90_temp/MonoInts/pot_ao_ints.irp.F90(102): error #6404: This name does not have a type, and must have an explicit type.   [DO_PSEUDO]
  if (do_pseudo.eqv..TRUE.) then                       ! MonoInts/pot_ao_ints.irp.f:  13
------^
compilation aborted for IRPF90_temp/MonoInts/pot_ao_ints.irp.F90 (code 1)
make[1]: *** [IRPF90_temp/MonoInts/pot_ao_ints.irp.o] Error 1
make[1]: *** Waiting for unfinished jobs....
make[1]: Leaving directory `/home/scemama/quantum_package/src/Bielec_integrals'
-------------------- Error --------------------

Build failed for module Bielec_integrals

-----------------------------------------------
make: *** [default] Error 255

```
Because MonoInts/EZFIO.cfg is does not rebuild do_pseudo in Bielec_integrals.
All EZFIO.cfg files should be transformed to *.irp.f files before.

**Comment by TApplencourt** [comment]
Strange, in my version is work perfectly.
I will investigate this.

Normaly after the `make veryclean` all the EZFIO.cfg are touch.
(Can you check the in `/scripts/module/clean_modules.sh` if it is the case?)

So after the new make
the `/scripts/ezfio_interface/prepare_ezfio.sh` is trigger.
This script convert the EZFIO.cfg into *.irp.f and cp in  EZFIO/config
Then the EZFIO compile.



2015-05-06 15:09 GMT+02:00 Anthony Scemama <notifications@github.com>:

**Comment by TApplencourt** [comment]
This is fix by #8dc7ea8481ba79094d2f073df9d0fd1f68f20d2e
